### DISCUSSION TITLE
What icon library does React Navigation default header use

### DISCUSSION THREAD

**Discussion by elmcapp** [react-navigation/react-navigation#10345] [Q&A] [answered]
### Current behavior

I am not sure what icon Library and icon names is used in the default header

### Expected behavior

I would like to know the name of the icon and library used for the default header buttons. I am building a custom header and would like to match the header button from the photos.
<img width="437" alt="Screen Shot 2022-02-13 at 1 15 20 AM" src="https://user-images.githubusercontent.com/64275658/153741587-91d2bc19-8560-4d9b-926e-652bb95abbef.png">
 

### Reproduction

na

### Platform

- [ ] Android
- [X] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [ ] @react-navigation/bottom-tabs
- [X] @react-navigation/drawer
- [ ] @react-navigation/material-bottom-tabs
- [ ] @react-navigation/material-top-tabs
- [ ] @react-navigation/stack
- [ ] @react-navigation/native-stack

### Environment

- [] I've removed the packages that I don't use

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               |
| @react-navigation/bottom-tabs          |
| @react-navigation/drawer               |
| @react-navigation/material-bottom-tabs |
| @react-navigation/material-top-tabs    |
| @react-navigation/stack                |
| @react-navigation/native-stack         |
| react-native-safe-area-context         |
| react-native-screens                   |
| react-native-gesture-handler           |
| react-native-reanimated                |
| react-native-tab-view                  |
| react-native-pager-view                |
| react-native                           |
| expo                                   |
| node                                   |
| npm or yarn                            |

**Comment by satya164** [comment, marked as answer]
It's an image https://github.com/react-navigation/react-navigation/blob/main/packages/drawer/src/views/assets/toggle-drawer-icon%404x.ios.png
